### PR TITLE
feature/iesp-122

### PR DESCRIPTION
**Pull request by acolella92**

### CODE REVIEW

**Review by nttdata-rtorsoli — DISMISSED** [review]

**Review by galales — DISMISSED** [review]
Fine by me, just a comment: with this implementation it's difficult to know how many and which parameters are required.
Instead of constants, messages could have been functions

File: src/main/java/it/pagopa/interop/probing/probingapi/exception/RestExceptionHandler.java [hunk 1]
```diff
@@ -15,66 +15,68 @@
...
+   * Creates an instance of type {@link Problem} following the RFC 7807 standard
+   *
+   * @param responseCode  The response error code
+   * @param titleMessage  The response title message
+   * @param detailMessage The response detail error message
+   * @return A new instance of {@link Problem}
+   */
+  private Problem createProblem(HttpStatus responseCode, String titleMessage,
+      String detailMessage) {
+    Problem genericError = new Problem();
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/probingapi/exception/RestExceptionHandler.java]: Use Builder (with Default), and Accessors feature of lombok

**Review by nttdata-rtorsoli — APPROVED** [review]

**Review by nttdata-rtorsoli — DISMISSED** [review]
See comments

File: src/main/java/it/pagopa/interop/probing/probingapi/exception/RestExceptionHandler.java (hunk 1 above)
Comment (acolella92) [reply src/main/java/it/pagopa/interop/probing/probingapi/exception/RestExceptionHandler.java]: change done and pushed

**Review by galales — APPROVED** [review]

**Review by nttdata-rtorsoli — APPROVED** [review]

**Pull request merged**